### ISSUE TITLE
A standalone example of Lit

### ISSUE THREAD

**Issue by gmosx** [lit/lit#1929]
## Description

Lit looks like a very promising framework, congrats!  However I cannot find a complete, stand-alone example that shows how to implement a trivial application (e.g. a todo list) using Lit.

## Acceptance criteria

The example should demonstrate best practices for both the development mode (e.g. hot reloads) but also in production (e.g. bundling)

**Comment by gmosx** [comment]
I have spoken to early, found this:

https://github.com/lit/lit/tree/main/packages/lit-starter-ts

**Comment by BrentLabasan** [comment]
Here is an example of a to do list made with LitElement 2.1 https://github.com/shprink/web-components-todo/tree/master/lit-element-2.1/src

**Comment by gmosx** [comment]
Thank you for the link.

This is an old example. Also I would like to see a more full-featured example, showing how to bundle for production, etc.

**Comment by mr-rampage** [comment]
@gmosx I may be wrong, so take this with a grain of salt, but I believe that the use case for lit is to extend the HTML specification. It can be used to make applications by composing web components, but it's not going to be the same as building with React or Angular. It's more of a library than a framework. That's not to say it's impossible to build SAP with lit, but it's probably not the use case.

The nice thing is that you can use lit with React or Angular.

**Comment by ajmchambers** [comment]
Not exactly a full app or anything, but these are pretty good reference for more of an app with routing etc:
- https://github.com/pwa-builder/pwa-starter
- https://github.com/IBM/pwa-lit-template

But yea I agree it would be nice to have more complete app-like examples.

**Comment by gmosx** [comment]
Thank you for the links. I am also working on something, when I have something concrete I might post it on a blog or github.

**Comment by abdonrd** [comment]
> Thank you for the links.
> However, they seem to use the older 2.x version of lit.
> 
> I am also working on something, when I have something concrete I might post it on a blog or github.

We have a work in progress PR with the update to Lit v2: https://github.com/IBM/pwa-lit-template/pull/202

**Comment by gmosx** [comment]
Interesting, thanks. Carbon Design System will also get updated to support the latest Lit version?

**Comment by abdonrd** [comment]
> Interesting, thanks. Carbon Design System will also get updated to support the latest Lit version?

Maybe @jeffchew can answer this.

**Comment by jeffchew** [comment]
@abdonrd it's been on our radar. Just opened an issue to track an investigation to upgrade: https://github.com/carbon-design-system/carbon-web-components/issues/657

**Comment by davidbwaters** [comment]
I agree that more examples should be added.  I assume this is planned and know this is the new version, but the existing example with eleventy is a very specific use case and doesn't really help a new user.  I think a stripped down MVP that shows the dev server setup would be most useful for people to adapt into whatever they are using.  

The docs recommend open-wc, but their starter still has the entire app as a component or is for sharing a component as a package.  I think there should starter for a PWA and an example of a component used in a static site, suitable for new users and people learning web dev.  I would be willing to help contribute to boilerplates.  Ideally, having starters for different scenarios with npx and `yarn create` would be a huge boost for adoption.  Then, ssr, with and without typescript, and other examples could be added.

A lot of React's popularity comes from `create-react-app` and not having to deal with tooling.

Put yourself in the shoes of someone learning and imagine having to comprehend all the tooling included in the current starter.

**Comment by davidbwaters** [comment]
Check out https://github.com/litelement-dev/lit-simple-starter-kit.  This is a great example of a stripped down starter using vite and outlines all the steps to generate it.  This could be adapted to use the modern-web server.
